instagram/src/Authentication/Authenticate.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+const Authenticate = (App, Login) =>
```

Comment:
Nice!

Reply:
Ah I thought I handled having no initial local storage! But maybe I just never tested it? I don't know but tweaked it and should work with a cleared localStorage now!